assets/js/main.js

```diff
@@ -0,0 +1,173 @@
+  // document.querySelector('#square' + squareTwo).style.background = 'black'
+  // document.querySelector('#square' + squareThree).style.background = 'black'
+  // document.querySelector('#square' + squareFour).style.background = 'grey'
+
+  // var squareOneStyle = document.querySelector('#square' + squareOne)
+  // squareOneStyle.style.backgroundImage = 'url("http://orig04.deviantart.net/7d15/f/2016/366/c/0/cat_head_png__by_madcatmd-datnw9e.png")'
+  // squareOneStyle.style.backgroundSize = '100%'
+  // squareOneStyle.style.backgroundRepeat = 'no-repeat'
+  // squareOneStyle.style.backgroundPosition = 'center'
+
+  var squareOneStyle = document.querySelector('#square' + gameArray[0])
+  squareOneStyle.style.backgroundImage = 'url("http://orig04.deviantart.net/7d15/f/2016/366/c/0/cat_head_png__by_madcatmd-datnw9e.png")'
+  squareOneStyle.style.backgroundSize = '100%'
+  squareOneStyle.style.backgroundRepeat = 'no-repeat'
+  squareOneStyle.style.backgroundPosition = 'center'
+
+
+  var squareTwoStyle = document.querySelector('#square' + squareTwo)
+  squareTwoStyle.style.backgroundImage = 'url("http://orig04.deviantart.net/7d15/f/2016/366/c/0/cat_head_png__by_madcatmd-datnw9e.png")'
+  squareTwoStyle.style.backgroundSize = '100%'
+  squareTwoStyle.style.backgroundRepeat = 'no-repeat'
+  squareTwoStyle.style.backgroundPosition = 'center'
+
+
+
+  var squareThreeStyle = document.querySelector('#square' + squareThree)
+  squareThreeStyle.style.backgroundImage = 'url("http://orig04.deviantart.net/7d15/f/2016/366/c/0/cat_head_png__by_madcatmd-datnw9e.png")'
+  squareThreeStyle.style.backgroundSize = '100%'
+  squareThreeStyle.style.backgroundRepeat = 'no-repeat'
+  squareThreeStyle.style.backgroundPosition = 'center'
+
+
+  var squareFourStyle = document.querySelector('#square' + squareFour)
+  squareFourStyle.style.backgroundImage = 'url("http://orig04.deviantart.net/7d15/f/2016/366/c/0/cat_head_png__by_madcatmd-datnw9e.png")'
+  squareFourStyle.style.backgroundSize = '100%'
+  squareFourStyle.style.backgroundRepeat = 'no-repeat'
+  squareFourStyle.style.backgroundPosition = 'center'
+}
+// ---------------------------------------------------------
+function buttons (event) {
+  // var numpad = 0
+  if (event.keyCode === 90) {
+    numpad = 9
+  } else if (event.keyCode === 88) {
+    numpad = 10
+  } else if (event.keyCode === 78) {
+    numpad = 11
+  } else if (event.keyCode === 77) {
+    numpad = 12
+  } else if (event.keyCode === 32) {
+    startpad = 1
+  }
+}
+// ---------------------------------------------------------
+function restart () {
+  gameArray = []
+  counter = 0
+  time = 30
+  fired = false
+  var allSquare = document.querySelectorAll('.square')
+  for (var i = 0; i < allSquare.length; i++) {
+    allSquare[i].style.background = 'white'
+  document.querySelector('.counter').innerHTML = '0'
+  document.querySelector('.time').innerHTML = '30'
+  }
+}
+// ---------------------------------------------------------
+var fired = false
```

Comment:
this variable should be placed on top too, with the rest of other global variables